lib/ansible/modules/identity/cyberark/cyberark_user.py

```diff
@@ -0,0 +1,515 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
```

Comment:
There should be a copyright line in here.  It can be as simple as 
```
# Copyright 2017 Ansible Project
```
if you want.  For any legal questions, we would have to use the git log to find out the real copyright holders so a catchall like above is okay.